Source/Engine/Config.hpp

```diff
@@ -12,10 +12,10 @@ class Config
         bool vulkanValidationEnabled = true;
 
         Window::Mode startUpWindowMode = Window::Mode::eWindowed;
-        vk::Extent2D defaultWindowExtent = vk::Extent2D{1280, 720};
+        vk::Extent2D defaultWindowExtent = vk::Extent2D{ 1280, 720 };
```

Comment:
Cpp11BracedListStyle option is responsible for space in curled braces.
Now it is set to false, but if to set it to true - bigger and more uglier diff is generated